api_data.py

```diff
@@ -1,34 +1,35 @@
 #!/usr/bin/env python
 
 import json
-from urllib import urlopen 
+import sys
 
-def output_cve_id(): 
+"""Py version check"""
+if sys.version_info[0] == 3:
+    from urllib.request import urlopen
+else: 
+	from urllib import urlopen 
+
+def output_cve_id(type=None, name=None, version=None): 
 	"""Takes as input, a package name, package version. 
 	Queries cve-search' dataset for any reported 
 	vulnerabilities of the requested package. If 
 	vulnerability exists, outputs cve-id(s).
 	"""
-	package_name = raw_input('Enter package name: ')
-	user_choice = raw_input('Do you have a package version? (Y/N): ')
-	
-	if user_choice == 'Y' or user_choice == 'y':
-		package_ver = raw_input('Enter package version: ')
-		url = 'https://cve.circl.lu/api/search/' + package_name + package_ver
-	
+	if version:
+		url = 'https://cve.circl.lu/api/search/' + name + version
```

Comment:
@pombredanne The name is a basic necessity. I was thinking of maybe keeping the whole function in a try-catch block, since if the name is none. We have nothing to search?